### PR TITLE
Add chatops section to boostrap script [ci skip]

### PR DESCRIPTION
**Pull request by enykeev**

### CODE REVIEW

File: scripts/st2bootstrap-deb.sh [hunk 1]
```diff
@@ -100,13 +106,50 @@ install_st2web() {
...
+  sudo apt-get install -y st2chatops
+}
+
+configure_st2chatops {
+  # Set credentials
+  sudo sed -i.bak -r "s/^(export ST2_AUTH_USERNAME.).*/\1$USERNAME/" /opt/stackstorm/chatops/st2chatops.env
+  sudo sed -i.bak -r "s/^(export ST2_AUTH_PASSWORD.).*/\1$PASSWORD/" /opt/stackstorm/chatops/st2chatops.env
+
+  # Setup adapter
+  if [ "$HUBOT_ADAPTER" == "slack" ] && [ -z "$HUBOT_SLACK_TOKEN" ]
```
Comment (enykeev) [inline scripts/st2bootstrap-deb.sh]: I wonder if we should even start this guessing game or just go straight to the warning. Without installer, we would have a very hard time trying to create proper user experience here.

Comment (lakshmi-kannan) [inline scripts/st2bootstrap-deb.sh]: This is fine for now. We can fix it later for other adapters.

**Comment by lakshmi-kannan** [comment]
lgtm.

File: scripts/st2bootstrap-deb.sh [hunk 2]
```diff
@@ -100,13 +106,50 @@ install_st2web() {
   sudo service nginx restart
 }
 
+install_st2chatops {
```
Comment (lakshmi-kannan) [inline scripts/st2bootstrap-deb.sh]: Should be function with  ()

File: scripts/st2bootstrap-deb.sh [hunk 3]
```diff
@@ -100,13 +106,50 @@ install_st2web() {
...
+  # Install Node
+  curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
+  sudo apt-get install -y nodejs
+
+  # Install st2chatops
+  sudo apt-get install -y st2chatops
+}
+
+configure_st2chatops {
```
Comment (lakshmi-kannan) [inline scripts/st2bootstrap-deb.sh]: Should be function with ()

File: scripts/st2bootstrap-deb.sh (hunk 1 above)
Comment (lakshmi-kannan) [inline scripts/st2bootstrap-deb.sh]: Should be ! -z "$HUBOT_SLACK_TOKEN"

**Comment by lakshmi-kannan** [comment]
Closing in favor of https://github.com/StackStorm/st2-packages/pull/206

**Pull request closed without merging**